docs/2-Collection_Containers.adoc

```diff
@@ -965,6 +965,64 @@ They are named *{IntHashSet}*, *{FloatHashSet}* etc.
 *{BooleanHashSet}* is implemented using a single integer to hold one of four possible states: ([], [F], [T], or [T, F]).
 All other sets use open addressing and quadratic probing.
 
+There are different ways you can create *IntHashSet*, which are listed as below.
+You can create an primitive sets using **newSetWith** method. You can initialize an *IntHashSet*  using constructor argument. There are 5 different overloaded constructor are available.
+Below examples are provided with *IntHashSet* but all are same for *FloatHashSet* and *BooleanHashSet* as well.
+[source,java]
+----
+// Initialize IntHashSet using newSetWith method, which is taking int vargs.
+IntHashSet intHashSet1 = IntHashSet.newSetWith(1, 2, 3, 4, 5);
```

Comment:
Change the types on the left to the interfaces `MutableIntSet` , `MutableFloatSet` and `MutableBooleanSet`.